apps/server/src/shared/domain/entity/news.entity.ts

```diff
@@ -61,14 +61,20 @@ export abstract class News extends BaseEntityWithTimestamps {
 	@ManyToOne(() => SchoolEntity, { fieldName: 'schoolId' })
 	school!: SchoolEntity;
 
-	@ManyToOne('User', { fieldName: 'creatorId' })
-	creator!: User;
+	@ManyToOne('User', { fieldName: 'creatorId', nullable: true })
+	creator?: User | undefined;
 
 	@ManyToOne('User', { fieldName: 'updaterId', nullable: true })
 	updater?: User;
 
 	permissions: string[] = [];
 
+	public removeCreatorReference(creatorId: EntityId): void {
```

Comment:
I see also updater: User - makes sense to remove also references of this?

Reply:
This ticket is only about deleting creatorId reference from news

Reply:
making that creator optional is kinda annoying though, not blocking this pr but maybe you have a better idea?

Reply:
every other ticket is done like that we are always setting the field to undefined

Reply:
as we don't want to make some kind of tombstone field

Reply:
@virgilchiriac we discussed it and we will add deletion of updater in terms of this PR.

Reply:
ok, +1 for consistency